rllib/algorithms/dqn/torch/default_dqn_torch_rl_module.py

```diff
@@ -154,8 +205,11 @@ def compute_advantage_distribution(
         # Reshape the action values.
         # NOTE: Handcrafted action shape.
         logits_per_action_per_atom = torch.reshape(
-            batch, shape=(-1, self.action_space.n, self.num_atoms)
+            batch, shape=(*batch.shape[:-1], self.action_space.n, self.num_atoms)
         )
+        # logits_per_action_per_atom = torch.reshape(
```

Comment:
remove?

Reply:
I check. I will run some tests, if all hyperparameter combis in DQN still run.

Reply:
Great catch! Done.